web-common/src/components/CellInspector.svelte

```diff
@@ -0,0 +1,167 @@
+<script lang="ts">
+  import { onMount } from "svelte";
+  import { createEventDispatcher } from "svelte";
+  import { cellInspectorStore } from "../features/dashboards/stores/cellInspectorStore";
+  import { formatInteger } from "../lib/formatters";
+
+  export let value: string = "";
+  export let isOpen: boolean = false;
+
+  const dispatch = createEventDispatcher();
+  let hovered = false;
+  let hoveredValue: string | null = null;
+  let container: HTMLElement;
+  let content: HTMLElement;
+
+  function handleKeyDown(event: KeyboardEvent) {
+    if (event.code === "Space" && !event.repeat) {
+      event.preventDefault();
+      event.stopPropagation();
+      dispatch("toggle");
+    } else if (event.key === "Escape") {
+      event.preventDefault();
+      event.stopPropagation();
+      if (isOpen) {
+        dispatch("close");
+      }
+    }
+  }
+
+  function handleClickOutside(event: MouseEvent) {
+    if (isOpen && container && !container.contains(event.target as Node)) {
+      dispatch("close");
+    }
+  }
+
+  onMount(() => {
+    window.addEventListener("keydown", handleKeyDown, true);
+    document.addEventListener("click", handleClickOutside, true);
+
+    // Handle hover events
+    const handleHover = (e: MouseEvent) => {
```

Comment:
This whole thing is unnecessary. We could just store the values in a global store in the event listeners in `useCellInspector`. (Storing globaly already seem to be happening, so this can be removed)